### ISSUE TITLE
Centos 8 Installation Problem

### ISSUE THREAD

**Issue by oguzhancik** [shiftkey/desktop#291]
### Operating System/Distribution

Which distribution of Linux are you using?

Centos 8.1

### Installer

Which version of the app?
2.5.2 Linux RC1

Which installer type?
RPM installer

### What happened?

When i try to install it, i see the following error:
libappindicator is needed by github-desktop-2.5.2.linux1-1.x86_64

Whenerver i looked the installed libraries i can see the corresponding library:
libappindicator-gtk3-0:12.10.0-19.el8.x86_64

Can you help for this issue ?

**Comment by shiftkey** [comment]
@oguzhancik thanks for the report. This dependency is something I believe is required for Electron apps and is documented here, and might not be available yet for CentOS 8.

https://github.com/shiftkey/desktop/blob/48d13371876db41480cd168862b69aa17052162d/script/package-redhat.ts#L44-L55

But that was from a while ago, and maybe it's no longer needed for Electron apps? I need to remember the command to check the shared libraries needed by a binary. Something like this script: https://stackoverflow.com/a/50218/1363815

**Comment by shiftkey** [comment]
I pulled out my Fedora distro and built the latest version from source to get that list:

```
$ yarn && yarn build:prod
...
$ ldd dist/github-desktop-linux-x64/github-desktop \
> | grep so \
> | sed -e '/^[^\t]/ d' \
> | sed -e 's/\t//' \
> | sed -e 's/.*=..//' \
> | sed -e 's/ (0.*)//' \
> | sort \
> | uniq -c \
> | sort -n
      1 /home/brendanforster/src/desktop/dist/github-desktop-linux-x64/libffmpeg.so
      1 /lib64/ld-linux-x86-64.so.2
      1 /lib64/libasound.so.2
      1 /lib64/libatk-1.0.so.0
      1 /lib64/libatk-bridge-2.0.so.0
      1 /lib64/libatspi.so.0
      1 /lib64/libavahi-client.so.3
      1 /lib64/libavahi-common.so.3
      1 /lib64/libblkid.so.1
      1 /lib64/libbz2.so.1
      1 /lib64/libcairo-gobject.so.2
      1 /lib64/libcairo.so.2
      1 /lib64/libcom_err.so.2
      1 /lib64/libcrypto.so.1.1
      1 /lib64/libc.so.6
      1 /lib64/libcups.so.2
      1 /lib64/libdatrie.so.1
      1 /lib64/libdbus-1.so.3
      1 /lib64/libdl.so.2
      1 /lib64/libepoxy.so.0
      1 /lib64/libexpat.so.1
      1 /lib64/libffi.so.6
      1 /lib64/libfontconfig.so.1
      1 /lib64/libfreetype.so.6
      1 /lib64/libfribidi.so.0
      1 /lib64/libgcc_s.so.1
      1 /lib64/libgcrypt.so.20
      1 /lib64/libgdk-3.so.0
      1 /lib64/libgdk_pixbuf-2.0.so.0
      1 /lib64/libgio-2.0.so.0
      1 /lib64/libglib-2.0.so.0
      1 /lib64/libgmodule-2.0.so.0
      1 /lib64/libgmp.so.10
      1 /lib64/libgnutls.so.30
      1 /lib64/libgobject-2.0.so.0
      1 /lib64/libgpg-error.so.0
      1 /lib64/libgraphite2.so.3
      1 /lib64/libgssapi_krb5.so.2
      1 /lib64/libgtk-3.so.0
      1 /lib64/libharfbuzz.so.0
      1 /lib64/libhogweed.so.5
      1 /lib64/libidn2.so.0
      1 /lib64/libk5crypto.so.3
      1 /lib64/libkeyutils.so.1
      1 /lib64/libkrb5.so.3
      1 /lib64/libkrb5support.so.0
      1 /lib64/liblz4.so.1
      1 /lib64/liblzma.so.5
      1 /lib64/libmount.so.1
      1 /lib64/libm.so.6
      1 /lib64/libnettle.so.7
      1 /lib64/libnspr4.so
      1 /lib64/libnss3.so
      1 /lib64/libnssutil3.so
      1 /lib64/libp11-kit.so.0
      1 /lib64/libpango-1.0.so.0
      1 /lib64/libpangocairo-1.0.so.0
      1 /lib64/libpangoft2-1.0.so.0
      1 /lib64/libpcre2-8.so.0
      1 /lib64/libpcre.so.1
      1 /lib64/libpixman-1.so.0
      1 /lib64/libplc4.so
      1 /lib64/libplds4.so
      1 /lib64/libpng16.so.16
      1 /lib64/libpthread.so.0
      1 /lib64/libresolv.so.2
      1 /lib64/librt.so.1
      1 /lib64/libselinux.so.1
      1 /lib64/libsmime3.so
      1 /lib64/libsystemd.so.0
      1 /lib64/libtasn1.so.6
      1 /lib64/libthai.so.0
      1 /lib64/libunistring.so.2
      1 /lib64/libuuid.so.1
      1 /lib64/libwayland-client.so.0
      1 /lib64/libwayland-cursor.so.0
      1 /lib64/libwayland-egl.so.1
      1 /lib64/libX11.so.6
      1 /lib64/libX11-xcb.so.1
      1 /lib64/libXau.so.6
      1 /lib64/libxcb-render.so.0
      1 /lib64/libxcb-shm.so.0
      1 /lib64/libxcb.so.1
      1 /lib64/libXcomposite.so.1
      1 /lib64/libXcursor.so.1
      1 /lib64/libXdamage.so.1
      1 /lib64/libXext.so.6
      1 /lib64/libXfixes.so.3
      1 /lib64/libXinerama.so.1
      1 /lib64/libXi.so.6
      1 /lib64/libxkbcommon.so.0
      1 /lib64/libXrandr.so.2
      1 /lib64/libXrender.so.1
      1 /lib64/libXss.so.1
      1 /lib64/libXtst.so.6
      1 /lib64/libz.so.1
      1 linux-vdso.so.1
```

I don't see any `libappindicator.so.*` libraries in here, which makes me think it's now unnecessary to include as a requirement.

**Comment by oguzhancik** [comment]
It looks like excluding libappindicator.so from rpm installer dependency check will solve the installation issue. I can try it with updated installer after you prepare it.

**Comment by shiftkey** [comment]
This should be resolved in [`2.5.3-linux1`](https://github.com/shiftkey/desktop/releases/tag/release-2.5.3-linux1)
